### DISCUSSION TITLE
Meal planner : share planning with visitor

### DISCUSSION THREAD

**Discussion by Draky50110** [mealie-recipes/mealie#4507] [Q&A] [answered]
Hello.

I have a private houseold/groupe but I enabled sharing of recipes for non group member.

I wish other people to see the menu planner without registering Mealie.
Is it possible to share the meal planner page, with receipe title or recipe content ?

Which settings need to be enabled ?

I don't want all visitors to see that as GoogleBot could scrape it :D

**Comment by michael-genson** [comment, marked as answer]
Currently there is no way to view the meal planner without logging in

**Comment by Draky50110** [comment]
Ok, thank you.
